rte/mo_rte_lw.F90

```diff
@@ -56,7 +57,7 @@ module mo_rte_lw
   function rte_lw(optical_props, top_at_1, &
                   sources, sfc_emis,       &
                   fluxes,                  &
-                  inc_flux, n_gauss_angles) result(error_msg)
+                  inc_flux, n_gauss_angles, type2stream) result(error_msg)
```

Comment:
Since we have only two choices here - two-stream or re-scaled - I would much prefer a logical variable here, like `use_2str` or similar. (No error checking required!) If the variable is not supplied we assume we will use the re-scaling solution.